src/pivot-table/hooks/use-column-width.ts

```diff
@@ -156,8 +181,16 @@ export default function useColumnWidth(
     qNoOfLeftDims,
     measureTextForHeader,
     estimateWidthForDimensionValue,
+    calculateLeftGridWidthInfo,
   ]);
 
+  const overrideLeftGridWidth = useCallback(
+    (width: number, index: number) => {
+      setLeftGridWidthInfo(calculateLeftGridWidthInfo(width, index));
+    },
+    [calculateLeftGridWidthInfo],
+  );
```

Comment:
probably you dont need useCallback in this case?

Reply:
I didn't have the callback first, and there was something complaining or not working. I'll look at it again

Reply:
that was for the `calculateLeftGridWidthInfo` function. So I think we could skip useCallback here. But I'd say it's better to take a complete look at this file. I think there is a overusage of useXxxx here, and we might want to break some things out as well. I think I prefer to keep this memoized, since we do it for everything else in this file. Especially the return values